test/__snapshots__/index.spec.js.snap

```diff
@@ -1701,7 +1701,39 @@ Array [
 
 exports[`should pass on fixtures/samples/es7/contrived.good.js 1`] = `Array []`;
 
-exports[`should pass on fixtures/samples/es7-object-spread/contrived.good.js 1`] = `Array []`;
+exports[`should pass on fixtures/samples/es7-object-spread/contrived.bad.js 1`] = `
+Array [
+  Object {
+    "column": 7,
+    "line": 7,
+    "rule": "no-unused-vars",
+    "severity": 2,
+  },
+  Object {
+    "column": 13,
+    "line": 7,
+    "rule": "prefer-object-spread/prefer-object-spread",
+    "severity": 2,
+  },
+]
+`;
+
+exports[`should pass on fixtures/samples/es7-object-spread/contrived.good.js 1`] = `
```

Comment:
This snapshot should have 0 errors. Simple use the variables in a `console.log` statement so that the errors go away. It also seems that there are some undeclared variables. Make sure that the `contrived.good` sample have 0 errors.